lib/app.js

```diff
@@ -9,7 +9,8 @@
 } (function (ko, Calculator, ToDoList) {
 
   function App() {
-    let calculators = ko.observableArray([new Calculator()]);
+    let calculators = ko.observableArray();
+
 
     function addCalculator() {
       calculators.push(new Calculator());
```

Comment:
I am wondering, why do you need to add `new Calculator()` for? When is that viewmodel ever used or bound to?